src/app/app.component.ts

```diff
@@ -14,6 +14,8 @@ import { Notifications } from '../providers/notifications';
   templateUrl: './app.html'
 })
 export class StockpileApp {
+  // Need to ViewChild Nav instead of using NavController because this is
```

Comment:
I think I actually do prefer `//` comments for inline comments inside functions. Since they are also used for one line comments that serve the same purpose as two or three line comments (like here), it would be confusing to switch the style. They also make it easier to differentiate between function description and explanation of smaller portion of code inside functions.